src/server.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+const express = require('express');
+const ex = express();
+const bodyParser = require('body-parser');
+var jsonParser = bodyParser.json();
+const cors = require('cors');
+const dbq = require('./queries.js');
+ex.use(cors());
+ex.use(express.static('./frontend'));
```

Comment:
On line 9, if we're not going to have a frontend folder, maybe we could remove this part from the file?